vello/src/scene/bitmap.rs

```diff
@@ -138,11 +140,32 @@ impl<'a> BitmapStrikes<'a> {
 #[derive(Clone)]
 enum StrikesKind<'a> {
     None,
-    Sbix(sbix::Sbix<'a>, GlyphMetrics<'a>),
+    Sbix(sbix::Sbix<'a>, SbixKind, GlyphMetrics<'a>),
     Cbdt(CbdtTables<'a>),
     Ebdt(EbdtTables<'a>),
 }
 
+#[derive(Copy, Clone, PartialEq)]
```

Comment:
Could this get a doc comment to indicate that it is for this particular workaround?

Reply:
Sure, added a note. Let me know if that is sufficient.

Reply:
Yep!